app/middleware/renderer.js

```diff
@@ -1,26 +1,29 @@
 const messages = require('../lib/messages');
-const postcodeValidator = require('../lib/postcodeValidator');
 const log = require('../lib/logger');
 
 function results(req, res) {
-  res.render('results');
+  if (res.locals.gps.length > 0) {
+    res.render('results');
+  } else {
+    res.render('no-results');
```

Comment:
Is there a search with no results which is not an error? If so, should we add a test?